### ISSUE TITLE
Tutorial feedback round 3

### ISSUE THREAD

**Issue by nsheff** [databio/bedhost#30]
- [x] Go down from 2 genomes to 1 to reduce external data size.
- [x] Don't write comments in code blocks. Use markdown blocks for text, code blocks for code.
- [x] what is 'CML' ?
- [x] please correct "(might have to do this before starting the tutorial)" comment
- [x] Quote below doesn't make sense to me. Where is the JSON file, in the configuration file? or is it for each bedset. I believe it should be a sample property, not a project property.

> We specify a name for the bedset and a database query term that will select which BED files should belong to the bedset. The PEP configuration file must also point to a JSON file describing an Elasticsearch query to select BED files of interest. 

FYI -- there was a this note:

>  If this hasn't been already done, we recommend starting this jupyter notebook enabling sudo permissions since steps such as downloading `docker` or running an elasticsearch `docker` container won't be executed otherwise. This can be done with `sudo jupyter notebook --allow-root`

I removed this because this is not necessary if you install docker correctly, adding the docker user to the sudoers group. this is outside the scope of this tutorial.
